### ISSUE TITLE
setup.py install doesn't work on Mac OS X again

### ISSUE THREAD

**Issue by qobilidop** [ipython-contrib/IPython-notebook-extensions#461]
@jonnybaik brought it up in #431 and resolved it in #432, but the issue seems to appear again. I was following the instructions in [README.md](https://github.com/ipython-contrib/IPython-notebook-extensions/blob/b124adca81c2c4e414b0487487eb91843d7d5b8a/README.md) to install the notebook extensions through conda. I checked out the master branch, then ran `conda build IPython-notebook-extensions`, but received the following error:

```
Traceback (most recent call last):
  File "/Users/qobilidop/miniconda3/envs/_build/lib/python3.5/site-packages/psutil/_psosx.py", line 199, in wrapper
    return fun(self, *args, **kwargs)
  File "/Users/qobilidop/miniconda3/envs/_build/lib/python3.5/site-packages/psutil/_psosx.py", line 238, in cmdline
    return cext.proc_cmdline(self.pid)
ProcessLookupError: [Errno 3] No such process

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 52, in <module>
    import configure_nbextensions
  File "/Users/qobilidop/miniconda3/conda-bld/work/configure_nbextensions.py", line 62, in <module>
    if ("python" or "jupyter") in p.name():
  File "/Users/qobilidop/miniconda3/envs/_build/lib/python3.5/site-packages/psutil/__init__.py", line 546, in name
    cmdline = self.cmdline()
  File "/Users/qobilidop/miniconda3/envs/_build/lib/python3.5/site-packages/psutil/__init__.py", line 599, in cmdline
    return self._proc.cmdline()
  File "/Users/qobilidop/miniconda3/envs/_build/lib/python3.5/site-packages/psutil/_psosx.py", line 209, in wrapper
    raise ZombieProcess(self.pid, self._name, self._ppid)
psutil.ZombieProcess: psutil.ZombieProcess process still exists but it's a zombie (pid=54)
```

which is extremely similar to the one in the previous issue #431.

But since the [`setup.py`](https://github.com/ipython-contrib/IPython-notebook-extensions/blob/b124adca81c2c4e414b0487487eb91843d7d5b8a/setup.py) looks completely different now after @jfbercher's commits, I don't know how to apply the previous solution here. Could anyone help?

My setup:
Mac OS X 10.11.3 (El Capitan)
Python 3.5.1, installed using conda

**Comment by jfbercher** [comment]
Hi @qobilidop,
Indeed, I missed this patch for Mac OS X  zombie processes (it was not implemented in the version I used as the basis for my modifications). The corresponding tests are now in `configure_nbextensions.py` -- we needed to split things in order to bukd a pip installable version. I think that the very same patch as in #432 should work. You will be welcome to submit this patch.

**Comment by qobilidop** [comment]
Thanks @jfbercher for the explanation and @baldwint for the solution!

**Comment by juhasch** [comment]
Fixed by #462
